consul_exporter.go

```diff
@@ -448,6 +464,7 @@ func main() {
 	kingpin.Flag("consul.server-name", "When provided, this overrides the hostname for the TLS certificate. It can be used to ensure that the certificate name matches the hostname we declare.").Default("").StringVar(&opts.serverName)
 	kingpin.Flag("consul.timeout", "Timeout on HTTP requests to the Consul API.").Default("500ms").DurationVar(&opts.timeout)
 	kingpin.Flag("consul.insecure", "Disable TLS host verification.").Default("false").BoolVar(&opts.insecure)
+	kingpin.Flag("consul.request-limit", "Limit the maximum number of concurrent requests to Consul.").Default("0").IntVar(&opts.requestLimit)
```

Comment:
this only covers the services api

Reply:
maybe specify that 0 = no limit

Reply:
> this only covers the services api

This is the only API endpoint that is called concurrently for the time being. If other endpoints were concerned in the future, I'd expect the same limit to be applied.

Reply:
good point

Reply:
done